mover-rclone/deployments/mysql/source/volumesnapshot-dynamic.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshot
+metadata:
+  name: scribe-snap
+spec:
+  volumeSnapshotClassName: gp2-csi
+  source:
+    persistentVolumeClaimName: mysql
```

Comment:
@cooktheryan I need to provide the source pvc name. To generalize it I am naming it to `source-pvc`